pycrostates/cluster/_base.py

```diff
@@ -747,6 +753,16 @@ def fitted(self, fitted):
             self._fitted_data = None
             self._fitted = False
```

Comment:
Add the reset for `self._labels_` in the base class (and remove from `fitted` setter in KMeans.